src/pages/Home/Home.consts.ts

```diff
@@ -14,8 +14,8 @@ const BUSINESS_VALUES = {
 
 const COMBAT_LEVEL_VALUES = {
   WARRIOR: 'לוחם',
-  COMBAT_SUPPORT: 'תומך_לחימה',
-  SUPPORT_ROLE: 'עורפי',
+  COMBAT_SUPPORT: 'תומך לחימה',
+  //SUPPORT_ROLE: 'עורפי',  // commented, not used
```

Comment:
Hey can you please explain why do we remove this option?

Reply:
as far as I can tell,  this  const and text is only defined here and not used in the frontend or backend for any display or logic

Reply:
You are right, looks like its not used in the FE.
@bendavidelad @itamargran do you expect to get it in the BE?

Reply:
Nope, we only use 'לוחם' or 'תומך לחימה'.
can be removed freely.